### PR TITLE
Add Liqoctl pipeline

### PR DESCRIPTION
**Pull request by palexster**
# Description

This PR adds the steps to build liqoctl and publish it with each release. It also fixes several step conditions which were broken and update the create release action, since the one currently used is deprecated.

# How Has This Been Tested?

- [x] Manually

### CODE REVIEW

**Review by giorio94 — APPROVED** [review]

File: .github/workflows/integration.yml [hunk 1]
```diff
@@ -186,9 +186,54 @@ jobs:
...
 
+  liqoctl:
+    runs-on: ubuntu-latest
+    needs: [ configure ]
+    strategy:
+      matrix:
+        goos: [ linux, windows, darwin ]
+        goarch: [ arm64, amd64 ]
+        exclude:
+          - goarch: arm64
```
[the remark this answers is not in the document]
Comment (palexster) [reply .github/workflows/integration.yml:197]: I read some outdated documentation and thought it was not supported. Adding it.

File: .github/workflows/integration.yml [hunk 2]
```diff
@@ -186,9 +186,54 @@ jobs:
...
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
+        env:
+          GOPATH: ${{ github.workspace }}
+
+      - uses: actions/checkout@v2
+        with:
+          path: ./liqo
```
[the remark this answers is not in the document]
Comment (palexster) [reply .github/workflows/integration.yml:213]: I had some issues with the GOPATH directory and the liqo clone being in the same directory.

[the remark this answers is not in the document]
Comment (giorio94) [reply .github/workflows/integration.yml:213]: Got it. You might also set the GOPATH to a temporary directory, the project does not need to be inside AFAIK

**Review by aleoli — APPROVED** [review]

**Pull request merged**